rust/gelato/src/task_status_call.rs

```diff
@@ -0,0 +1,117 @@
+use crate::err::GelatoError;
+use serde::{Deserialize, Serialize};
+use std::sync::Arc;
+use tracing::{info, instrument};
+
+const RELAY_URL: &str = "https://relay.gelato.digital";
```

Comment:
is this the same for mainnet and testnets?

Reply:
Seemingly, yes. We could ask Ed, but it has worked with all testnets I've tried, and it seems to transparently be the case when reading the relay sdk typescript implementation